sources/packages/backend/apps/api/src/route-controllers/assessment/assessment.controller.service.ts

```diff
@@ -88,8 +94,13 @@ export class AssessmentControllerService {
       );
     }
 
+    const assessmentDTO = assessment.assessmentData as AssessmentAPIOutDTO;
```

Comment:
we can do it like this without force casting. 

```ts
const assessmentDTO: AssessmentAPIOutDTO = assessment.assessmentData;
    if (maskTotalFamilyIncome) {
      assessmentDTO.totalFamilyIncome = MASKED_MONEY_AMOUNT;
    }
```

Reply:
The cast is necessary to allow it to accept a string.
![image](https://github.com/user-attachments/assets/13fe4993-7009-46dc-948c-08b4d1eebe13)

Reply:
Changed as suggested.